tests/models/qwen2_vl/test_modeling_qwen2_vl.py

```diff
@@ -245,6 +246,91 @@ def test_initialization(self):
                         msg=f"Parameter {name} of model {model_class} seems not properly initialized",
                     )
 
+    @pytest.mark.generate
+    def test_generate_continue_from_past_key_values(self):
```

Comment:
If the missing `rope_deltas` is the only issue, can we instead:
1 - replace `prepare_config_and_inputs_for_common()` by `prepare_config_and_inputs_for_generate()` in the base test
2 - Overload `prepare_config_and_inputs_for_generate` in this model tester, to also return `rope_deltas`

(much smaller code diff -> easier to maintain)

Reply:
Hmm, but we would need access to the model outputs from prev step in that case, which we don't have because `generate()` returns only some kwargs. And if we want to feed the original inputs with cache to the `prepare_config_and_inputs_for_generate`, we'll need to prepare `repo_deltas` again in the same way

Reply:
(chatted on slack about exploring how to standardize `qwen2_vl` + `generate`)